src/components/organize/TagsManager/TagDialog/ColorPicker.tsx

```diff
@@ -0,0 +1,73 @@
+import ReplayIcon from '@material-ui/icons/Replay';
+import { useIntl } from 'react-intl';
+import { Box, InputAdornment, TextField } from '@material-ui/core';
+
+import { DEFAULT_TAG_COLOR, hexRegex, randomColor } from '../utils';
+
+const ColorPicker: React.FunctionComponent<{
```

Comment:
Love this component - with the random colour button x
Who wants to click open an RGB picker and have to search through millions of possible shades? Life is too short....